src/Tizen.TV.UIControls.Forms/ContentCreator.cs

```diff
@@ -0,0 +1,129 @@
+
+            var view = new AbsoluteLayout();
+            view.Children.Add(image, new XForm.Rectangle(0, 0, 1, 1), AbsoluteLayoutFlags.All);
+            view.Children.Add(new StackLayout
+            {
+                VerticalOptions = LayoutOptions.EndAndExpand,
+                HorizontalOptions = LayoutOptions.FillAndExpand,
+                Children = {
+                    new StackLayout {
+                        VerticalOptions = LayoutOptions.EndAndExpand,
+                        HorizontalOptions = LayoutOptions.FillAndExpand,
+                        Padding = 15,
+                        Spacing = 0,
+                        BackgroundColor = XForm.Color.FromHex("#2b7c87"),
+                        Children = { text, detailLabel }
+                    }
+                }
+            }, new XForm.Rectangle(0, 0, 1, 1), AbsoluteLayoutFlags.All);
+
+            cell.SetBinding(GridView.BindingContextProperty, new Binding("BindingContext", source: view));
+            return view;
+        }
+
+        static View CreateContent(TextCell cell)
+        {
+            XForm.Label text = new XForm.Label
+            {
+                HorizontalTextAlignment = TextAlignment.Start,
+                VerticalTextAlignment = TextAlignment.Start,
+                HorizontalOptions = LayoutOptions.FillAndExpand,
+                VerticalOptions = LayoutOptions.FillAndExpand,
+                FontAttributes = FontAttributes.Bold,
+            };
+            XForm.Label detailLabel = new XForm.Label
+            {
+                HorizontalTextAlignment = TextAlignment.Start,
+                VerticalTextAlignment = TextAlignment.Start,
+                HorizontalOptions = LayoutOptions.FillAndExpand,
+                VerticalOptions = LayoutOptions.FillAndExpand,
+            };
+            detailLabel.FontSize = Device.GetNamedSize(NamedSize.Micro, detailLabel);
+
+            text.SetBinding(XForm.Label.TextProperty, new Binding("Text", source: cell));
+            text.SetBinding(XForm.Label.TextColorProperty, new Binding("TextColor", source: cell));
+
+            detailLabel.SetBinding(XForm.Label.TextProperty, new Binding("Detail", source: cell));
+            detailLabel.SetBinding(XForm.Label.TextColorProperty, new Binding("DetailColor", source: cell));
+
+            var view = new AbsoluteLayout();
+            view.Children.Add(new StackLayout
+            {
+                VerticalOptions = LayoutOptions.Center,
+                HorizontalOptions = LayoutOptions.FillAndExpand,
+                Children = {
+                    new StackLayout {
+                        VerticalOptions = LayoutOptions.Center,
+                        HorizontalOptions = LayoutOptions.FillAndExpand,
+                        Padding = 15,
+                        Spacing = 0,
+                        BackgroundColor = XForm.Color.FromHex("#2b7c87"),
+                        Children = { text, detailLabel }

```

Comment:
If `detailLabel` doesn't be added because `DetailText` is empty. `TVUIControl` can not support what text of `detailLabel` is set later. So, I would like to recommend that user uses `ViewCell` with single text, if developer want to use `TextCell` as single text style. Could you share your opinion?

Reply:
If you want to support `Cell`, it would be good to use the "text" and swallow parts (e.g. icon, end....) provided according to the native Gengrid item style.

Currently, both Cell and View are allowed in DataTemplate, but in fact View can actually cover in ViewCell, and in the case of TextCell, instead of using text part, create views/layout (in forms' layer) and put it in swallow.icon part.

In my personal opinion, I think it may not be necessary to support Cell, but if you want to support Cell, it would be better to implement it using the proper part defined in Gengrid style, not like the current implementation.